ods_ci/tasks/Resources/Provisioning/GPU/NVIDIA/gpu_install.yaml

```diff
@@ -27,7 +27,6 @@ spec:
   name: gpu-operator-certified
   source: certified-operators
   sourceNamespace: openshift-marketplace
-  startingCSV: "gpu-operator-certified.v1.11.1"
```

Comment:
could you check if https://github.com/red-hat-data-services/ods-ci/blob/master/ods_ci/tasks/Resources/Provisioning/GPU/NVIDIA/gpu_deploy.sh#L12 would create problems? maybe we can just remove that line from the script now

I don't know why it was put there in the first place, maybe @tarukumar remembers?

Reply:
I suggest to completely omit the version specification - in gpu_install.yaml as well as in gpu_deploy.sh

Reply:
@bdattoma I have noticed that the `sed` is incorrect, notice the version diffs
```
sed -i'' -e "0,/v1.11/s//$CHANNEL/g" -e "s/gpu-operator-certified.v1.11.0/$CSVNAME/g"  "$GPU_INSTALL_DIR/gpu_install.yaml"
```

vs

```
startingCSV: "gpu-operator-certified.v1.11.1"
```

Reply:
ok, are you going to remove the sed part in this PR?

Reply:
The fixed version was implemented because the the operator was causing the problme was causing issues if the starting CSV version wasn't provided. If this problem no longer occurs, we can remove this .

Reply:
I have removed the `sed` part and now I'm going to re-execute the installation with these changes.

Reply:
Hi @bdattoma @tarukumar I have re-executed the job together with a basic GPU test in disconnected as well as in online env and it works